setup.py

```diff
@@ -436,7 +439,7 @@ def do_setup():
         long_description_content_type='text/markdown',
         license='Apache License 2.0',
         version=version,
-        packages=find_packages(exclude=['tests*']),
+        packages=find_packages(exclude=['tests*', 'airflow/www/node_modules/*']),
```

Comment:
Should we also doc artifacts?

Reply:
Not really - I added it as tests and reverted it at the end. Those exclusions and inclusions are added in MANIFEST.in